plugins/imagebase/plugin.js

```diff
@@ -104,6 +272,46 @@
 			var widget = editor.widgets.add( name, createWidgetDefinition( editor, definition ) );
 
 			editor.addFeature( widget );
+		},
+
+		/**
+		 * Adds new feature to the newly created image widget by invoking initial setup once for the editor
```

Comment:
Well to be fair, you made it in a way it's compatible with any widget. So I suppose you can refer just to "widget" rather than "image widget" in this method docs.

Reply:
> (...) feature to the newly created image widget by (...)

This part kinda leads me to thinking that it's feature added for a single widget instance. Assumption which is cleared later on.

But I think we should make it clear that it's for a widget definition from the very beginning.